### ISSUE TITLE
Aggregation API

### ISSUE THREAD

**Issue by jmcarp** [18F/feedback-widget#1]
EOM

**Comment by rypan** [comment]
@jmcarp @afeld is there a slack channel for the crew? i'm a single channel guest...currently in DAP...i could switch out to the new one if there is one.

**Comment by afeld** [comment]
@rypan No, we could create one, though I think each of us is working on it infrequently enough that async over GitHub Issues is probably fine. Feel free to open new issues to start new conversations!

**Comment by jmcarp** [comment]
Also, it sounds like we're going to talk about this tomorrow at the 18F documentation working group. We can update the issue and talk about next steps afterwards. Meanwhile, the prototype is actually working now, although it probably still needs a bit more polish to call it an MVP.

**Comment by rypan** [comment]
@afeld let's keep it here then :) 

@jmcarp awesome - i hope the meeting goes well. when you have a sense of what issues/tasks are needed to go from where we are right now to MVP...add those...and we can all pitch in.
